extensions/exchanges/binance/exchange.js

```diff
@@ -0,0 +1,256 @@
+const ccxt = require('ccxt')
+  , path = require('path')
+  , colors = require('colors')
+  , moment = require('moment')
+  , _ = require('lodash')
+  , n = require('numbro')
+
+module.exports = function container (get, set, clear) {
+  var c = get('conf')
+
+  var public_client, authed_client
+
+  function publicClient () {
+    if (!public_client) public_client = new ccxt.binance({ 'apiKey': '', 'secret': '' })
+    return public_client
+  }
+
+  function authedClient () {
+    if (!authed_client) {
+      if (!c.binance || !c.binance.key || !c.binance.key === 'YOUR-API-KEY') {
```

Comment:
`if (!c.binance || !c.binance.key || !c.binance.key === 'YOUR-API-KEY') {`

This should be:
`if (!c.binance || !c.binance.key || c.binance.key === 'YOUR-API-KEY') {`

Because if `c.binance.key === 'YOUR-API-KEY'` we should stop